### PR TITLE
3.0 preview dev

### PR DESCRIPTION
**Pull request by keanulee**

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -25,12 +25,10 @@
...
 
-    const $_documentContainer = document.createElement('template');
-
-    $_documentContainer.innerHTML =
-        `<custom-style><style is="custom-style" include="demo-pages-shared-styles"></style></custom-style>`;
-
-    document.body.appendChild($_documentContainer.content);
+    import {html} from '@polymer/polymer/lib/utils/html-tag.js';
+    const template = html
+    `<custom-style><style is="custom-style" include="demo-pages-shared-styles"></style></custom-style>`;
```
Comment (e111077) [inline demo/index.html]: You can put the backtick on the line above so you can get syntax highlighting

Comment (e111077) [reply demo/index.html]:
```js

Comment (e111077) [reply demo/index.html]:
```js
html`
<custom-style>...</custom-style>`;
```

Comment (keanulee) [reply demo/index.html]: I already made this change in multiple elements - don't think it's necessary to go through all these repos again just for this change

Comment (e111077) [reply demo/index.html]: Sure, but you can still make the change here

**Pull request merged**